src/plrmodes/youmu_b.c

```diff
@@ -246,7 +246,10 @@ TASK(youmu_orb_homing_spirit_expire, { BoxedProjectile p; }) {
 }
 
 static int youmu_orb_homing_spirit_timeout(Projectile *orb) {
-	return orb->timeout - projectile_time(orb);
+	if(orb) {
+		return orb->timeout - projectile_time(orb);
+	}
+	return 0;
 }
```

Comment:
Adding `attr_nonnull(1)` before the function declaration would be a better idea - this should never be called with a NULL argument.

Reply:
I added `attr_nonnull(1)` and the compiler still spits out the same warning.

https://github.com/StarWitch/taisei/pull/1/checks?check_run_id=1606225668

https://github.com/StarWitch/taisei/pull/1/commits/5ffa51b633bf3dc94667008dc9798c314c0e0a61#diff-505af192e6781de8f60b076c8d9095e18d58d47b771836184907aa22f7efefc2R248

Reply:
The compiler sucks, then. `NOT_NULL(orb)` should shut it up, or `assume(orb != NULL);` before the return statement.